solidity/dashboard/src/pages/liquidity/LiquidityPage.jsx

```diff
@@ -51,6 +61,40 @@ const LiquidityPage = ({ headerTitle }) => {
 
   return (
     <PageWrapper title={headerTitle} newPage={true}>
+      <ClosableContainer
+        className={"tile liquidity-banner"}
+        hide={isActive || gt(keepTokenBalance.value, 0)}
+      >
+        <div className={"liquidity-banner__keep-logo"}>
```

Comment:
Could we use a `Banner` component? eg.

```
<Banner className="liquidity-banner">
 <div className="flex row">
  <Banner.Icon icon={Icons.KeepGreenOutline} />
  <div>
   <Banner.Title>Don’t yet have KEEP tokens?</Banner.Title>
   <Banner.Description className="text-secondary">
    What are you waiting for? KEEP can be bought on the open market
    on&nbsp;
    <a
     target="_blank"
     rel="noopener noreferrer"
     href={"https://balancer.exchange/#/swap"}
     className="text-link"
    >
     Balancer
    </a>
    &nbsp;or&nbsp;
    <a
     target="_blank"
     rel="noopener noreferrer"
     href={"https://app.uniswap.org/#/swap"}
     className="text-link"
    >
     Uniswap
    </a>
   </Banner.Description>
  </div>
 </div>
 <Banner.CloseIcon />
</Banner>
```

Reply:
If you agree with comment above then we need to update the `CloasableContainer` or rid of it.